Dockerhub-README.md

```diff
@@ -0,0 +1,10 @@
+# Running a2d2 on docker
+
+1. Use following docker build command in a2d2 
+
+	    docker build -t a2d2 . --target final
```

Comment:
We dont need docker build command in dockerhub readme, since we will be only using this readme file to run the a2d2